docker/domserver/scripts/start.d/50-domjudge.sh

```diff
@@ -82,9 +82,10 @@ fi
 # Add the Docker gateway as a trusted proxy
 if grep -q TRUSTED_PROXIES webapp/.env.local > /dev/null 2>&1
 then
-	sed -i "s|TRUSTED_PROXIES=.*|TRUSTED_PROXIES=${DOCKER_GATEWAY_IP}|" webapp/.env.local
+	PRE_TRUSTED_PROXIES=$(grep "TRUSTED_PROXIES" webapp/.env.local | sed 's|TRUSTED_PROXIES=||g')
```

Comment:
This will make it that every time you start the container it will keep adding the trusted proxy over and over again, won't it?

Reply:
Yes, what you said makes sense.

So we should remove `PRE_TRUSTED_PROXIES`.